far/FarEng.hlf.m4

```diff
@@ -3920,8 +3921,20 @@ the list. In this list "Brief mode" item corresponds to brief panel mode
 on. The last item, "Alternative full", corresponds to view mode called with
 #LeftCtrl+0#. After selecting the mode, you can change the following settings:
 
-  - #Column types# - column types are encoded as one or several
-characters, delimited with commas. Allowed column types are:
+  - #Column types# - column types are encoded as comma-delimited list
+of column specifiers. Each column specifier starts with
+a single-character denoting a file attribute. Some file attribute
```

Comment:
`file attribute` is not the best term here (and below) - it can be confused with "real" file attributes (Read Only, Hidden etc.).